pages.pt_BR/common/git-remote.md

```diff
@@ -0,0 +1,28 @@
+# git remote
+
+> Gerencia repositórios monitorados ("remotes").
```

Comment:
Nice!

Reply:
Gostei disso: usar o termo em inglês à partir de uma descrição dele no início.

Reply:
:)